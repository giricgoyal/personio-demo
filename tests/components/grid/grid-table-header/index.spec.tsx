import React from 'react'
import GridTableHeader from 'src/components/grid/grid-table-header'
import { render } from 'tests/test-utils'

describe('components/grid/grid-table-header', () => {
    let container

    const columnDef = [
        {
            title: 'Name',
            propBinding: 'name',
            filter: true,
            sort: true,
            type: 'string',
        },
    ]

    beforeEach(() => {
        ;({ container } = render(<GridTableHeader columnDef={columnDef} />))
    })

    test('should render and match snapshot', () => {
        expect(container).toMatchSnapshot()
    })
})
