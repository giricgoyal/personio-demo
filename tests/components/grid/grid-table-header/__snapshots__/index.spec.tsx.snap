// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/grid/grid-table-header should render and match snapshot 1`] = `
<div>
  <div
    class="grid__table__cell grid__table__cell--header"
  >
    Name
  </div>
</div>
`;
